CMakeLists.txt

```diff
@@ -41,7 +41,13 @@ include(../../cmake_modules/standard_setup.cmake)
 #==================================================================================================#
 set(VaultSourcesDir ${PROJECT_SOURCE_DIR}/src/maidsafe/vault)
 
+ms_glob_dir(DataManager ${VaultSourcesDir}/data_manager "Data Manager")
```

Comment:
missing "\\\\". may be we can get rid of it from other personas if its not required